src/ingest_validation_tools/table-schemas/antibodies.yaml

```diff
@@ -0,0 +1,33 @@
+fields:
+  # NOTE: Example had "dataset_uuid": Won't work. Not known at submission time.
+  -
+    name: antibody_name
+    description: TODO
+  -
+    name: rrr_id
+    description: TODO
+    # TODO: pattern constraint
+  -
+    name: uniprot_accession_number
+    description: TODO
+    # TODO: pattern constraint
+  -
+    name: lot_number
+    description: TODO
+  -
+    name: validation_report_location
+    description: TODO
+    # Is this part of the submission? Or a URL? If a URL, put that in the field name.
+  -
+    name: assay
+    description: TODO
```

Comment:
```suggestion
    description: This antibody.tsv is unique to a specific HuBMAP assay dataset. Enter the name of the specific assay using this antibody preparation in this field. If the antibody prep was used for multiple assays, the metadata should be re-entered on one row per assay.
```

Reply:
This goes to the data-modelling issue... Need to talk/clarify.

Reply:
@cebriggs7135 : Can you rephrase this as a description of what the field means? We want a description that could be used downstream... instructions for how to fill it in are not what the data consumer needs.

Reply:
moot